server_installation/topics/operator/installation.adoc

```diff
@@ -153,3 +153,10 @@ endif::[]
 
 * For more information on OpenShift Operators, see the link:https://docs.openshift.com/container-platform/4.4/operators/olm-what-operators-are.html[OpenShift Operators guide].
 
+[[_operator_prod_usage]]
+=== Using the {project_operator} in production environment
+
+* The usage of embedded DB is not supported in production environment.
```

Comment:
add "a" before "production"
change "prod" to "production"

Reply:
you mean "prod" in the chapter key : "_operator_prod_usage" ?

Reply:
:+1: